Header header
Point2D center
uint32 width
uint32 length
